applications/CompressiblePotentialFlowApplication/python_scripts/potential_flow_solver.py

```diff
@@ -167,6 +167,8 @@ def GetDefaultParameters(cls):
             "skin_parts":[""],
             "assign_neighbour_elements_to_conditions": false,
             "no_skin_parts": [""],
+            "transonic_scheme_settings" :{
```

Comment:
```suggestion
            "scheme_settings" :{
```
I'd call it `scheme_settings` for the sake of generality (imagine that in the future we require another one)